lib/bolt/executor.rb

```diff
@@ -4,18 +4,26 @@
 require 'bolt/result'
 require 'bolt/config'
 require 'bolt/notifier'
-require 'bolt/node'
 require 'bolt/result_set'
+require 'bolt/transport/ssh'
+require 'bolt/transport/winrm'
+require 'bolt/transport/orch'
 
 module Bolt
   class Executor
-    attr_reader :noop
+    attr_reader :noop, :transports
     attr_accessor :run_as
 
     def initialize(config = Bolt::Config.new, noop = nil, plan_logging = false)
       @config = config
       @logger = Logging.logger[self]
 
+      @transports = {
```

Comment:
previously we didn't initialize transports until we used them.

Reply:
I suppose this does defeat the purpose of moving the require statements inside the initialize methods... I'll update.